<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogOptions</class>
 <widget class="QDialog" name="DialogOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="General">
    <attribute name="title">
     <string>General</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Impression">
    <attribute name="title">
     <string>Impression</string>
    </attribute>
    <widget class="QGroupBox" name="gBFormatImpression">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>220</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>           Format d'impression</string>
     </property>
     <widget class="QRadioButton" name="rBA4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>A4</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBA3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>A3</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Multijoueur">
    <attribute name="title">
     <string>Multijoueur</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="pBOk">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBCancel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBApply">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
